### PR TITLE
Add formatting rule for type init in style guide

### PR DESCRIPTION
**Pull request by NipunaMarcus**
## Purpose
This will update the style guideline to have a rule for type init and fix some typos in the examples.

### CODE REVIEW

File: website/two-column-pages/docs/learn/style-guide/expressions.md [hunk 1]
```diff
@@ -206,4 +206,20 @@ table<Employee> employee3 = table {
     ]
 }
 ```
+
+## Type initialization
+
+* `new` keyword should always be followed by a single space.
+
+  Example,
```
Comment (anupama-pathirage) [inline website/two-column-pages/docs/learn/style-guide/expressions.md]: Shall we make the example bold here? It is bold in other places.

File: website/two-column-pages/docs/learn/style-guide/expressions.md [hunk 2]
```diff
@@ -206,4 +206,20 @@ table<Employee> employee3 = table {
...
+  ```ballerina
+    http:Response res = new ();
+  ```
+  
+* If the type name is available, there should be no spaces between the opening parentheses and the type name.
+
+  Example,
```
Comment (anupama-pathirage) [inline website/two-column-pages/docs/learn/style-guide/expressions.md]: Shall we make the example bold here? It is bold in the other places.

File: website/two-column-pages/docs/learn/style-guide/expressions.md (hunk 1 above)
Comment (NipunaMarcus) [reply website/two-column-pages/docs/learn/style-guide/expressions.md]: Add the requested change with latest commit.

File: website/two-column-pages/docs/learn/style-guide/expressions.md (hunk 2 above)
Comment (NipunaMarcus) [reply website/two-column-pages/docs/learn/style-guide/expressions.md]: Add the requested changes with latest commit.

**Pull request merged**